spiketools/spatial/occupancy.py

```diff
@@ -75,17 +97,35 @@ def compute_spatial_bin_assignment(position, x_edges, y_edges, include_edge=True
     >>> position = np.array([[1.5, 2.5, 3.5, 5], [6.5, 7.5, 8.5, 9]])
     >>> x_edges = np.array([1, 2, 3, 4, 5])
     >>> y_edges = np.array([6, 7, 8, 9, 10])
-    >>> x_bins, y_bins = compute_spatial_bin_assignment(position, x_edges, y_edges)
+    >>> compute_spatial_bin_assignment(position, x_edges, y_edges)
+    (array([1, 2, 3, 4]), array([1, 2, 3, 4]))
+
+    Compute bin assignment of 1d position, given existing 1d spatial bins:
+    >>> position = np.array([1.5, 2.5, 3.5, 5])
+    >>> x_edges = np.array([1, 2, 3, 4, 5])
+    >>> compute_spatial_bin_assignment(position, x_edges)
+    array([1, 2, 3, 4])
     """
 
-    x_bins = np.digitize(position[0, :], x_edges, right=False)
-    y_bins = np.digitize(position[1, :], y_edges, right=False)
+    # 2d case
+    if (y_edges is not None):
```

Comment:
For consistency across function, I would test this based on dimensionality of `positions`.